src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -927,7 +927,7 @@ def get_all_zonegroups_info(self):
             all_zonegroups_info['default_zonegroup'] = ''  # type: ignore
         return all_zonegroups_info
 
-    def delete_zonegroup(self, zonegroup_name: str, delete_pools: str):
+    def delete_zonegroup(self, zonegroup_name: str, delete_pools: str, pools: List[str]):
```

Comment:
```suggestion
    def delete_zonegroup(self, zonegroup_name: str, delete_pools: bool, pools: List[str]):
```
Let's try to fix this delete_pool str closer to the endpoint so that we don't propagate stringified booleans into deeper layers.